### ISSUE TITLE
Rendered emoji in JSON body response shifts line

### ISSUE THREAD

**Issue by aedelbro** [a85/POSTMan-Chrome-Extension#6150]
**Describe the bug**
When an emoji is included in the response body, the line containing that emoji is shifted down by a few pixels.

**To Reproduce**
Hit an endpoint that responds with an emoji on a line. Use the pretty formatter for JSON
ie.
```JSON
{
  "payload": {
    "data": [
      {
        "accountId": 1,
        "accountName": "Thrift Bank \ud83d\udd25",
        "count": 1,
        "totalMinutes": 30
      },
      {
        "accountId": 2,
        "accountName": "Chez Quis",
        "count": 2,
        "totalMinutes": 60
      }
    ]
  }
}
```

**Expected behavior**
The line that contains the emoji should be equidistant from lines on either side of it, and the indent indicators shouldn't have a gap.

**Screenshots**
Here is a screenshot of the above response body:
![image](https://user-images.githubusercontent.com/36162221/54950044-ce44c900-4f16-11e9-9e27-4d4f4fe4dea3.png)
[Alternative imgur link](https://i.imgur.com/R0SpcIe.png) 

**App information:**
 - App Type: Native App for Mac
 - Postman Version: 6.7.1 (6.7.1)
 - OS: macOS Mojave 10.14.3

**Comment by kamalaknn** [comment]
This should be fixed in the latest version of Postman above 7.3. Please update and give it a try.

**Comment by aedelbro** [comment]
Resolved with 7.3.4 (I also like the theming update too)
![Screen Shot 2019-07-24 at 9 52 36 AM](https://user-images.githubusercontent.com/36162221/61799396-0ee71d00-adf9-11e9-81d7-72ac0b1e0eff.png)
